.github/ISSUE_TEMPLATE/---feature-request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "\U0001F680 Feature request"
+about: Suggest an idea for Lemmur
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+### Is your feature request related to a problem? Please describe.
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
>Ex.

shouldn't that be "e.g."?

Reply:
This was in the default GitHub issue template